aleo-setup1/test.sh

```diff
@@ -20,7 +20,3 @@ cargo run --release --bin phase1 -- --curve-kind $CURVE --batch-size $BATCH --po
 
 yes | cargo run --release --bin phase1 -- --curve-kind $CURVE --batch-size $BATCH --power $SIZE contribute --challenge-fname challenge3 --response-fname response3
 cargo run --release --bin phase1 -- --curve-kind $CURVE --batch-size $BATCH --power $SIZE verify-and-transform --challenge-fname challenge3 --response-fname response3 --new-challenge-fname challenge4
-
-# add a randomness contribution by a beacon at the end
-cargo run --release --bin phase1 -- --curve-kind $CURVE --batch-size $BATCH --power $SIZE beacon --challenge-fname challenge4 --response-fname response4
```

Comment:
After discussing it with some people, I'm thinking that we should perhaps leave the beacon in.